libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.ts

```diff
@@ -231,6 +231,15 @@ export class SkyTextEditorComponent
     return this.#_toolbarActions;
   }
 
+  @Input()
+  public set openLinksInNewWindowOnly(value: boolean) {
```

Comment:
Instead of a boolean property that controls one option, use a property like `linkWindowOptions` that takes an array of allowed options. Make the default value include both options, and when the input is set to `undefined`, set it back to the default. Follow the pattern used by `toolbarActions` which takes an array of strings that match the union type of all the valid options.

String union type:
```
export type SkyTextEditorLinkWindowOptionType = 'new' | 'existing';
```

Default value const (in `text-editor.component.ts` defined above the class):
```
const LINK_WINDOW_OPTIONS_DEFAULT: SkyTextEditorLinkWindowOptionsType[] = ['new', 'existing'];
```

Component input:
```
@Input()
public set linkOptions(value: SkyTextEditorLinkWindowOptionType[] | undefined) {
  this.#_linkOptions = value ?? LINK_WINDOW_OPTIONS_DEFAULT;
}

public get linkOptions(): SkyTextEditorLinkWindowOptionsType[] {
  return this.#_linkOptions;
}

#_linkOptions = LINK_WINDOW_OPTIONS_DEFAULT;
```

Reply:
Done!